src/Carousel.tsx

```diff
@@ -43,11 +43,17 @@ export function Carousel({
 		scrollSliderPrevious(slider);
 	}
 
+	function uuid() {
+		return (
+			Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15)
+		);
+	}
+
 	return (
 		<div className="carousel">
 			<div ref={slider} className="carousel__slider">
-				{children.map((child, index) => (
-					<div key={index} className="carousel__slide">
+				{children.map((child) => (
+					<div key={uuid()} className="carousel__slide">
```

Comment:
Based on [this](https://stackoverflow.com/a/62631521), should not we avoid calling the `uuid()` function while returning the component?

Reply:
you're right, in this case we could use a simple implementation, like this:
```javascript
{children.map((child, index) => (
	<div key={`carouseItem${index}`} className="carousel__slide">
		{child}
	</div>
))}
```
interpolate the index list with a generic string